core/src/main/java/com/scalar/db/transaction/common/AbstractTwoPhaseCommitTransactionManager.java

```diff
@@ -9,51 +9,27 @@
 import com.scalar.db.api.Scan;
 import com.scalar.db.api.TwoPhaseCommitTransaction;
 import com.scalar.db.api.TwoPhaseCommitTransactionManager;
-import com.scalar.db.config.DatabaseConfig;
 import com.scalar.db.exception.transaction.CommitException;
 import com.scalar.db.exception.transaction.CrudException;
 import com.scalar.db.exception.transaction.PreparationException;
 import com.scalar.db.exception.transaction.RollbackException;
 import com.scalar.db.exception.transaction.TransactionException;
 import com.scalar.db.exception.transaction.UnknownTransactionStatusException;
 import com.scalar.db.exception.transaction.ValidationException;
-import com.scalar.db.util.ActiveExpiringMap;
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 import javax.annotation.Nullable;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public abstract class AbstractTwoPhaseCommitTransactionManager
     implements TwoPhaseCommitTransactionManager {
 
-  private static final long TRANSACTION_EXPIRATION_INTERVAL_MILLIS = 1000;
-
-  private static final Logger logger =
-      LoggerFactory.getLogger(AbstractTwoPhaseCommitTransactionManager.class);
-
   private Optional<String> namespace;
   private Optional<String> tableName;
 
-  private final ActiveExpiringMap<String, TwoPhaseCommitTransaction> activeTransactions;
```

Comment:
Ditto. Removed the active transaction management things from `AbstractTwoPhaseCommitTransactionManager`.